package.json

```diff
@@ -28,7 +28,7 @@
     "devDependencies": {
         "@prettier/plugin-xml": "^0.7.2",
         "@salesforce/eslint-config-lwc": "^0.5.0",
-        "@salesforce/sfdx-lwc-jest": "^0.7.1",
+        "@salesforce/sfdx-lwc-jest": "^0.8.0",
```

Comment:
That's not enough, you've also to run an npm install, so that the `package-lock.json` gets updated.